JavaScript/NuGet.JavaScript/NuGet.JavaScript.csproj

```diff
@@ -39,8 +42,17 @@
       <Link>snippet.html</Link>
       <CopyToOutputDirectory>PreserveNewest</CopyToOutputDirectory>
     </Content>
+    <None Include="packages.config" />
```

Comment:
<Content Include="packages.config">?

Reply:
@jpiyali a few lines above, packages.nuspec is also used as a None element. Do you think I should still change this one?

Reply:
Actually, they should both be none. We dont need these to be copied to output. No changes required.